src/content/en/updates/2018/12/get-installed-related-apps.md

```diff
@@ -77,17 +75,10 @@ either not prompting to install the PWA, or providing different prompts.
 | 1. Create explainer                          | [Complete][explainer]        |
 | **2. Create initial draft of specification** | [**In Progress**][spec]      |
 | **3. Gather feedback & iterate on design**   | [**In Progress**](#feedback) |
-| **4. Origin trial**                          | [**In Progress**](#ot)       |
-| 5. Launch                                    | Not started                  |
+| **4. Origin trial**                          | Complete                     |
+| 5. Launch                                    | In beta                      |
 
-### See it in action
```

Comment:
I played around with the heading here because I thought I was going to remove the origin trial. Then I remembered that the origin trial will still be ongoing when the post-ot version hits beta. I left the new structure because it's what we'll need to do anyway when the OT ends. If you review these changes using GitHub's split diff rather than it's unified diff, it will make more sense.